app/controllers/omniauth_callbacks_controller.rb

```diff
@@ -1,11 +1,13 @@
 class OmniauthCallbacksController < Devise::OmniauthCallbacksController
+  
+  skip_load_and_authorize_resource
+  
   def facebook
-    # You need to implement the method below in your model (e.g. app/models/user.rb)
     @user = User.from_omniauth(request.env["omniauth.auth"])
 
     if @user.persisted?
-      sign_in_and_redirect @user, :event => :authentication #this will throw if @user is not activated
-      set_flash_message(:notice, :success, :kind => "Facebook") if is_navigational_format?
+      sign_in_and_redirect @user, event: :authentication #this will throw if @user is not activated
```

Comment:
common practice is to put a space after the # symbol for readability. just looks cleaner